pkg/runtime/optimize/dml/select.go

```diff
@@ -148,7 +148,7 @@ func optimizeSelect(ctx context.Context, o *optimize.Optimizer) (proto.Plan, err
 
 	if flag&_bypass != 0 {
 		if len(stmt.From) > 0 {
-			err := rewriteSelectStatement(ctx, stmt, o)
+			err := expandSelectStar(ctx, stmt, o)
```

Comment:
why refactor the function name?

Reply:
this function is used to expand sql like `select * from student` to `select a,b from student`, so named with expandSelectStar  is more accurate.